ci-operator/step-registry/ipi/conf/azure/bootdiagnostics/ipi-conf-azure-bootdiagnostics-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "controlPlane boot diagnostics type: ${AZURE_CONTROL_PLANE_BOOT_DIAGNOSTICS_TYPE}"
+echo "Compute boot diagnostics type: ${AZURE_COMPUTE_BOOT_DIAGNOSTICS_TYPE}"
+echo "DefaultMachinePlatform boot diagnostics type: ${AZURE_DEFAULT_MACHINE_BOOT_DIAGNOSTICS_TYPE}"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [[ -f "${SHARED_DIR}/azure_storage_account_name" ]] && [[ -f "${SHARED_DIR}/resourcegroup_sa" ]]; then
+    sa_name=$(< "${SHARED_DIR}/azure_storage_account_name")
+    sa_resource_group=$(< "${SHARED_DIR}/resourcegroup_sa")
```

Comment:
maybe we can rename `resourcegroup_sa` to `sa_resourcegroup` for easy reading ?

Reply:
Do you mean the file name `resourcegroup_sa`? This is generated in step [azure-provision-storage-account](https://github.com/openshift/release/blob/master/ci-operator/step-registry/azure/provision/storage-account/azure-provision-storage-account-commands.sh#L23-L24) which also used to hypershift related jobs., I just read that file here.

Reply:
okay, to avoid the affect, then let it go.